Add link to API docs (#598)
returntocorp/semgrep-docs: docs/semgrep-app/semgrep-web-app.md

---
slug: semgrep-web-app
append_help_link: true
description: >-
  This document links to Semgrep API documentation.
---

# Semgrep Web App API

> Semgrep Web App is a Semgrep App API that enables you to list deployments, gather findings created by Semgrep App, and list projects. The API documentation uses **OpenAPI format**. The API requires a [teams tier account](https://semgrep.dev/pricing/), with the subsequent API token provisioned in your settings.

Author: ```suggestion
Semgrep App provides an API that enables you to list deployments, gather findings created by Semgrep App, and list projects. The API documentation uses the **OpenAPI format**. The API requires a [Team or Enterprise tier account](https://semgrep.dev/pricing/), with the subsequent API token provisioned in your settings.
```

> Semgrep Web App is a Semgrep App API that enables you to list deployments, gather findings created by Semgrep App, and list projects. The API documentation uses **OpenAPI format**. The API requires a [teams tier account](https://semgrep.dev/pricing/), with the subsequent API token provisioned in your settings.

Author: Implemented!
